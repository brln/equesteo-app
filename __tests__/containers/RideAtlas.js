import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import RideAtlasContainer from '../../containers/RideAtlas'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {
      rideAtlasEntries: {}
    },
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <RideAtlasContainer />
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})

