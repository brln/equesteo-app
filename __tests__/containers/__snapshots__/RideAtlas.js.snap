// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    You have no atlas entries. To view another map while you're riding, save it to your Atlas from the 'Tools' menu on that ride.
  </Text>
</View>
`;
